resource_alks_ltk.go

```diff
@@ -78,7 +78,9 @@ func resourceAlksLtkRead(d *schema.ResourceData, meta interface{}) error {
 
 	log.Printf("[INFO] alks_ltk.id: %v", d.Id())
 
-	return populateResourceDataFromLTK(resp, d)
+	_ = d.Set("iam_username", resp.UserName)
+
```

Comment:
I think you should be setting as much information here as possible so that the `read` operation fulfills its contract.  So basically, add in the `access_key`.  In the future we should consider returning additional information (like the ARN) back to the user when the ALKS LTK endpoint is called.